cluster-api/src/main/java/com/sequenceiq/cloudbreak/cluster/service/ClusterComponentConfigProvider.java

```diff
@@ -29,25 +22,42 @@
 import com.sequenceiq.cloudbreak.common.type.ComponentType;
 import com.sequenceiq.cloudbreak.domain.stack.cluster.Cluster;
 import com.sequenceiq.cloudbreak.domain.stack.cluster.ClusterComponent;
+import com.sequenceiq.cloudbreak.domain.stack.cluster.ClusterComponentHistory;
 import com.sequenceiq.cloudbreak.domain.view.ClusterComponentView;
+import com.sequenceiq.cloudbreak.repository.ClusterComponentHistoryRepository;
 import com.sequenceiq.cloudbreak.repository.ClusterComponentRepository;
 import com.sequenceiq.cloudbreak.repository.ClusterComponentViewRepository;
 import com.sequenceiq.cloudbreak.util.VersionNormalizer;
+import org.apache.commons.codec.binary.Base64;
```

Comment:
the import order is _still_ incorrect. please update your format for jJava as attached in the project under `config/idea/idea_settings.jar`